### DISCUSSION TITLE
V3 mockup (Figma+screenshots)

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcasite#104] [Ideas]
2022 versions of the mockup for version 3 of the live-listening app

[Figma link](https://www.figma.com/file/k0lM6w4muG2MMTDdGARSW6/Orcasite-%2F-Orcasound-Mobile-%26-Desktop-Listening-App-Redesign?node-id=3095%3A24138)

<img width="1525" alt="Screenshot 2023-02-01 at 4 30 22 PM" src="https://user-images.githubusercontent.com/14044595/216201164-787c4139-4f43-4778-91b7-ca232dabc9c0.png">

**Comment by scottveirs** [comment]
Hey @UXBrendan -- Today you mentioned in the standup today that the v3 UI for subscriptions had been specified in Figma. I think you mean this element in the upper righthand corner for desktop mockup, right?

<img width="573" alt="Screenshot 2023-07-12 at 2 46 24 PM" src="https://github.com/orcasound/orcasite/assets/14044595/f193c265-0e1c-4de6-8f5f-96af2d06feef">

And this is what's currently mocked-up for mobile, right?

<img width="1006" alt="Screenshot 2023-07-12 at 2 49 07 PM" src="https://github.com/orcasound/orcasite/assets/14044595/54201cff-e02e-41ea-a472-0ab98e3edee3">

If I got this wrong, could you link to the relevant Figma pages and maybe also drop screenshots here to be clear about which UI specification is the latest/preferred?

**Reply by UXBrendan** [reply]
@scottveirs 

Correct- those are the buttons/links to sign up for notifications. We rewrote the copy to a single phrase "Notify Me" and made it a button on desktop/tablet, but a more descriptive link at the bottom of mobile.

**Reply by UXBrendan** [reply]
@scottveirs 

Also, the notification sign up link is included in the global navigation, so should be on every content page on orcasound.net and every landing page for all web apps. We're still discussing how navigation functions for web apps one level down from the landing page.

**Reply by UXBrendan** [reply]
Here is the link for the design sent to development: https://www.figma.com/file/YdvbAjHETokV2x1LBHaElj?node-id=837:1026&mode=design#496522205

 
![Orcahome sent to development](https://github.com/orcasound/orcasite/assets/65036138/6292946a-5e6d-40d9-a6d1-13cc475d2e3a)
